utils/anautilities.py

```diff
@@ -442,6 +442,59 @@ def filePathExists(searchPath, subPath=None, debug=False):
             print "Found %s"%s(testPath)
         return True
 
+#========================================================================================
+# Find Inflection Point /////////////////////////////////////////////////////////////////
+#----------------------------------------------------------------------------------------
+# Author(s): Brendan Regnery, Brian Dorney, Jared Sturdy ////////////////////////////////
+#----------------------------------------------------------------------------------------
```

Comment:
these comments should follow the sphinx style syntax.  The best way I think is to incorporate yourself into the `moduleauthor` field at the top of the `RST` block at the top of the file rather than the "per function" level.